model-optimizer/extensions/front/tf/CTCGreedyDecoderReplacement.py

```diff
@@ -53,11 +56,12 @@ def run_after(self):
     @staticmethod
```

Comment:
please update the comment for this transformation. It is no longer valid since we save original format for sequence length.